ci/checks/style.sh

```diff
@@ -13,41 +13,77 @@ LANG=C.UTF-8
 # Activate common conda env
 source activate gdf
 
-# Run isort and get results/return code
-ISORT=`isort --check-only python/**/*.py`
-ISORT_RETVAL=$?
+# Run isort-cudf and get results/return code
+ISORT_CUDF=`isort python/cudf --check-only --settings-path=python/cudf/setup.cfg --skip-glob *.pyx --skip-glob *.pyi 2>&1`
```

Comment:
For some reason, the pre-commit hooks for `isort` automatically skip Cython files, while the CLI does not, so I had to specify to skip them here. Do we want package sorting for Cython files?

Reply:
Looks like we can get Cython/PYI resorting in pre-commit's isort hook with:

```yaml
types: [text]
types_or: [python, cython, pyi]
```

So if we would like package resorting on Cython it's now totally doable.